trusted/rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "nightly-2022-04-29"
```

Comment:
Consider switching to the simpler non-TOML syntax, since we're not specifying anything beyond `channel`

Reply:
maybe we should use the toml file so that we don't have to mimic `init_debian.sh` in the main mobilecoin repo
```
rustup component add \
  clippy \
  llvm-tools-preview \
  rust-analysis \
  rust-src \
  rustfmt
```
those could go into
```toml
components = [...]
```

Reply:
Started specifying clippy and rustfmt in future changes so resolving this